galerautils/src/gu_config.hpp

```diff
@@ -84,7 +84,7 @@ class gu::Config
         else
         {
 #ifndef NDEBUG
-            log_error << key << " not recognized.";
+            log_debug << "Key '" << key << "' not recognized.";
 #endif
```

Comment:
Yes, this is a message to debug the parser, here's how it looked before:
```
2015-04-23 17:53:00.236 ERROR: "gmcast.listen_addr not recognized.
2015-04-23 17:53:00.236 ERROR: Unrecognized parameter '"gmcast.listen_addr'
```
So here we are losing the first line.